spinta/backends/mongo.py

```diff
@@ -0,0 +1,102 @@
+import contextlib
+
+from spinta.commands import load, prepare, migrate, check, push, get, getall, wipe
+from spinta.components import Context, BackendConfig, Manifest, Model
+from spinta.backends import Backend
+
+
+class Mongo(Backend):
+    # Instance of this class will be created when application starts. First if
+    # will be loaded from configuration using `load` command, then it will be
+    # prepared from manifest declarations using `prepare` command.
+    #
+    # Backend also must have a `transaction` method which must return read or
+    # write transaction object containing an active `connection` to database.
+    metadata = {
+        'name': 'mongo',
+        'properties': {
+            'dsn': {'type': 'string', 'required': True},
+        },
+    }
+
+    @contextlib.contextmanager
+    def transaction(self, write=False):
+        with self.engine.begin() as connection:
+            if write:
+                # TODO: get a real transaction id
+                transaction_id = 1
```

Comment:
If you are not going to implement this now, maybe create a task.

Reply:
Created a task in our internal JIRA, as discussed.